src/main/java/mate/academy/hibernate/relations/dao/impl/ActorDaoImpl.java

```diff
@@ -12,7 +14,12 @@ public ActorDaoImpl(SessionFactory sessionFactory) {
 
     @Override
     public Actor add(Actor actor) {
-        return null;
+        Session session = factory.openSession();
+        Transaction transaction = session.beginTransaction();
+        session.save(actor);
+        transaction.commit();
+        session.close();
+        return actor;
```

Comment:
Lets use try-catch-finally block here, so we can rollback transaction in case if smth went wrong(like on the video in this topic https://mate.academy/learn/java-hibernate/java-entities-and-transactions/theory?section=theory). Same in other classes